<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Taufiq Marie</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_43">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea_menu">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="needcss" stdset="0">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>930</width>
           <height>580</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_36">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTabWidget" name="tabWidget_home">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>140</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>140</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="documentMode">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="tab_home">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <attribute name="title">
                <string>Home</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_41">
                <item row="0" column="3">
                 <layout class="QGridLayout" name="gridLayout_14">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_non_duplicate">
                    <property name="title">
                     <string>Nomor Non-duplicate</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_22">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_18">
                       <item row="0" column="0">
                        <widget class="QToolButton" name="toolButton_non_duplicate">
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset resource="resource.qrc">
                           <normaloff>:/res/system-software-install.png</normaloff>:/res/system-software-install.png</iconset>
                         </property>
                         <property name="iconSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="4">
                 <layout class="QGridLayout" name="gridLayout_15">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_data_provider">
                    <property name="title">
                     <string>Data Provider</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_17">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_16">
                       <item row="0" column="0">
                        <widget class="QToolButton" name="toolButton_data_provider">
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset resource="resource.qrc">
                           <normaloff>:/res/system-software-install.png</normaloff>:/res/system-software-install.png</iconset>
                         </property>
                         <property name="iconSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_2">
                    <property name="title">
                     <string>Database</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_13">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_7">
                       <item row="0" column="0">
                        <widget class="QToolButton" name="toolButton_database">
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset resource="resource.qrc">
                           <normaloff>:/res/folder-documents.png</normaloff>:/res/folder-documents.png</iconset>
                         </property>
                         <property name="iconSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="popupMode">
                          <enum>QToolButton::MenuButtonPopup</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="2">
                 <layout class="QGridLayout" name="gridLayout_10">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_data_master">
                    <property name="title">
                     <string>Data Master</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_12">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_11">
                       <item row="0" column="0">
                        <widget class="QToolButton" name="toolButton_data_master">
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset resource="resource.qrc">
                           <normaloff>:/res/system-software-install.png</normaloff>:/res/system-software-install.png</iconset>
                         </property>
                         <property name="iconSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="6">
                 <layout class="QGridLayout" name="gridLayout_19">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_reset_nomor">
                    <property name="title">
                     <string>Reset Data Nomor</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_21">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_20">
                       <item row="0" column="0">
                        <widget class="QToolButton" name="toolButton_reset_data_nomor">
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset resource="resource.qrc">
                           <normaloff>:/res/edit-delete.png</normaloff>:/res/edit-delete.png</iconset>
                         </property>
                         <property name="iconSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="popupMode">
                          <enum>QToolButton::MenuButtonPopup</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="5">
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_pengaturan">
                    <property name="title">
                     <string>Pengaturan</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_9">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_8">
                       <item row="0" column="0">
                        <widget class="QToolButton" name="toolButton_pengaturan">
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset resource="resource.qrc">
                           <normaloff>:/res/preferences-desktop-keyboard.png</normaloff>:/res/preferences-desktop-keyboard.png</iconset>
                         </property>
                         <property name="iconSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="1">
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_import_csv">
                    <property name="title">
                     <string>Input Data CSV</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_4">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_3">
                       <item row="0" column="0">
                        <widget class="QToolButton" name="toolButton_upload_ms_access">
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset resource="resource.qrc">
                           <normaloff>:/res/system-software-update.png</normaloff>:/res/system-software-update.png</iconset>
                         </property>
                         <property name="iconSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="popupMode">
                          <enum>QToolButton::MenuButtonPopup</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="7">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="TabWidgetUtama" name="tabWidget_utama">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="tabPosition">
               <enum>QTabWidget::North</enum>
              </property>
              <property name="tabShape">
               <enum>QTabWidget::Rounded</enum>
              </property>
              <property name="elideMode">
               <enum>Qt::ElideMiddle</enum>
              </property>
              <property name="documentMode">
               <bool>true</bool>
              </property>
              <property name="tabsClosable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionKeluar">
   <property name="text">
    <string>keluar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidgetUtama</class>
   <extends>QTabWidget</extends>
   <header location="global">tabwidgetutama.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionKeluar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
